lib/parser.ts

```diff
@@ -4,19 +4,25 @@ enum Source {
   WebDL = 'webdl',
 }
 
-interface Result {
+type Result = {
   show: string;
-  episodeCount: number;
   season: number;
-  episode?: number;
   year?: number;
   name?: string;
   codec?: string;
   quality?: number;
   source?: Source;
   group?: string;
   ext?: string;
-}
+} & (
+  | {
```

Comment:
This pipe is only to make it look pretty right? Not needed for typescript? (Not well versed in it to be honest).